.github/workflows/build-llvm.yml

```diff
@@ -0,0 +1,72 @@
+name: Build with LLVM against MSVCRT and Windows SDK
+
+on:
+  push:
+    branches: [ master ]
+    tags: [ v* ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  build-llvm:
+    runs-on: ubuntu-latest
+    container: docker.io/archlinux/archlinux:base
+
+    steps:
+    - name: Install packages
+      run: pacman -Syu --needed --noconfirm clang curl git lld llvm meson ninja
+
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        submodules: recursive
+        fetch-depth: 0
+
+    - name: Suppress repository ownership mismatch errors
+      run: git config --global --add safe.directory '*'
+
+    - name: Prepare MSVCRT and Windows SDK
+      run: |
+        xwin_version=0.6.6-rc.2
```

Comment:
We could use something like this to always fetch the latest version since they seem to release quite often.

```
curl -s https://api.github.com/repos/Jake-Shadle/xwin/releases/latest | jq  -r '.tag_name'
```

Reply:
On one hand, autoupdating this would be nice, but on another we kinda have no guarantees of stable behavior.

But as least I'll know when another release that fixes casing of `include` and `lib` is made because my hack below will likely break :sweat_smile:

Reply:
Done, hopefully in a way that shouldn't break in the future…

Reply:
Yeah, there is no perfect solution. If it breaks on annoying moment, we can still pin to a specific version in the future.